pkg/dogstatsd/parser.go

```diff
@@ -71,8 +82,17 @@ func parseTags(rawTags []byte, extractHost bool, defaultHostname string) ([]stri
 	var tag []byte
 	for {
 		tag, remainder = nextField(remainder, tagSeparator)
-		if extractHost && bytes.HasPrefix(tag, []byte("host:")) {
-			host = string(tag[5:])
+		if bytes.HasPrefix(tag, hostTagPrefix) {
+			host = string(tag[lenHostTagPrefix:])
+		} else if bytes.HasPrefix(tag, entityIDTagPrefix) {
+			// currently only supported for pods
+			entity := kubelet.KubePodPrefix + string(tag[lenEntityIDTagPrefix:])
+			entityTags, err := getTags(entity, tagger.DogstatsdCardinality)
+			if err != nil {
+				log.Tracef("Cannot get tags for entity %s: %s", entity, err)
```

Comment:
Should it happen very often? Could it be a `log.Debugf`?

Reply:
@remeh yes it can happen especially as we cannot guarantee the value of the environment variable, a wrong setting could make this quite spammy that's why I'd rather avoid the `debug` level.
We have other tools on the tagger (like `tagger-list`) to investigate potentials tagging issues

Reply:
👍